### PR TITLE
Movie create

### PR DESCRIPTION
**Pull request by tieupham2803**
movie list

### CODE REVIEW

**Comment by tieupham2803** [comment]
ready

File: resources/views/backend/movies/list.blade.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+        @if (session('success'))
+            <div class="alert alert-success">
+                {{ session('success') }}
+            </div>
+        @endif
+
+        <div class="panel-body">
+            <table class="table table-striped table-bordered table-hover" id="dataTables">
+                    <thead>
```
Comment (BunnyPi04) [inline resources/views/backend/movies/list.blade.php]: tab lại

File: resources/views/backend/movies/list.blade.php [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+@endsection
+@section('script')
+    <script>
+        $(document).ready( function () {
+            $('#dataTables').DataTable({
+                'order': [[ 0, 'desc' ]],
+            });
+        });
+        function xacnhanxoa(msg){
+            if(window.confirm(msg)){
```
Comment (BunnyPi04) [inline resources/views/backend/movies/list.blade.php]: convention

**Comment by BunnyPi04** [comment]
commented

**Pull request merged**